templates/netlify/app/entry.client.tsx

```diff
@@ -1,4 +1,4 @@
 import { RemixBrowser } from "@remix-run/react";
-import { hydrate } from "react-dom";
+import { hydrateRoot } from "react-dom/client";
 
-hydrate(<RemixBrowser />, document);
+hydrateRoot(document, <RemixBrowser />);
```

Comment:
```suggestion
hydrate(<RemixBrowser />, document);
```

Reply:
@MichaelDeBoey, we're using React 18 because we're using streaming. cc: @ascorbic

Reply:
Just for some additional context, @jacob-ebey moved us to v18 in our Edge template. See https://github.com/netlify/remix-edge-template/commit/787e25b57aef42d28470ab3af64fca45e3acbcde

Reply:
@jacob-ebey I assume React v18 isn't _really_ necessary & we can keep using v17 for now?

All other templates are using v17, so I think we want to have this consistent.

Once we update to v18, we'll of course change this back to how it implemented currently.

Reply:
I'm going to go ahead and revert this to v17 for now because of https://twitter.com/ryanflorence/status/1526278116527640576, [#2947](https://github.com/remix-run/remix/issues/2947) and https://github.com/facebook/react/issues/24430